nautobot/docs/development/core/kubernetes-job-support.md

```diff
@@ -0,0 +1,412 @@
+
+```bash
+NAME               DATA   AGE
+dev-env            32     24s
+nautobot-cm1       1      24s
+nautobot-cm2       1      24s
+```
+
+To start all persistent volume claims:
+
+```bash
+kubectl apply -f "development/kubernetes/*-persistentvolumeclaim.yaml"
+```
+
+You should see the following output:
+
+```bash
+persistentvolumeclaim/media-root created
+persistentvolumeclaim/pgdata-nautobot created
+```
+
+You can confirm that those persistent volume claims are created by running the following command:
+
+```bash
+kubectl get persistentvolumeclaims
+```
+
+You should see the following output:
+
+```bash
+NAME              STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   VOLUMEATTRIBUTESCLASS   AGE
+media-root        Bound    pvc-011f484b-2ccf-4fe0-953b-289a13ad0480   200Mi      RWO            standard       <unset>                 2m35s
+pgdata-nautobot   Bound    pvc-5954eb3f-75e3-4f6b-9b9c-a91e40ea96bf   200Mi      RWO            standard       <unset>                 2m35s
+```
+
+To confirm all required kubernetes entities are up and running, run the following command:
+
+```bash
+kubectl get all
+```
+
+You should see the following output, note that pods are automatically created when we create deployments:
+
+```bash
+NAME                               READY   STATUS    RESTARTS      AGE
+pod/celery-beat-6fb67477b7-rsw62   1/1     Running   0             30m
+pod/db-8687b48964-gtvtc            1/1     Running   0             30m
+pod/nautobot-679bdc765-pl2ld       1/1     Running   0             30m
+pod/redis-7cc58577c-tl5sq          1/1     Running   0             30m
+
+NAME                 TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
+service/db           ClusterIP   10.111.0.30     <none>        5432/TCP   30m
+service/kubernetes   ClusterIP   10.96.0.1       <none>        443/TCP    30m
+service/nautobot     ClusterIP   10.106.32.53    <none>        8080/TCP   30m
+service/redis        ClusterIP   10.102.99.143   <none>        6379/TCP   30m
+
+NAME                          READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/celery-beat   1/1     1            1           30m
+deployment.apps/db            1/1     1            1           30m
+deployment.apps/nautobot      1/1     1            1           30m
+deployment.apps/redis         1/1     1            1           30m
+```
+
+### Port Forward to Local Host
+
+We can use the `port-forward` command from `kubectl` to make your Nautobot instance on the kubernetes cluster accessible in `localhost:8080`:
+
+```bash
+kubectl port-forward <nautobot-pod-name> 8080:8080
+```
+
+The output should be:
+
+```bash
+Forwarding from 127.0.0.1:8080 -> 8080
+Forwarding from [::1]:8080 -> 8080
+Handling connection for 8080
+```
+
+Now go to your web browser and navigate to `localhost:8080`. You should see your Nautobot instance running.
+
+![K8s Nautobot Login](../../img/kubernetes/k8s_nautobot_login.png)
```

Comment:
Our image files in the docs are in several different directories already but I'd rather we not add more files to `docs/img`. Would you mind moving these to `docs/media/development/core/kubernetes/` instead?